### PR TITLE
Fix healthchecker

### PR DESCRIPTION
**Pull request by bzhaoopenstack**
1. Add service error status in openlabcmd
2. Remove built-in libiary logging
3. Refacor to fit new zk API.
4. Fix some issues in healthchecker scripts.

### CODE REVIEW

File: playbooks/ha_healthchecker.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
     - name: install script dependency
       pip:
-        name: ['six', 'iso8601', 'requests', 'configparser', 'PyGithub', 'openlabcmd', 'logging']
+        name: ['six', 'iso8601', 'requests', 'configparser', 'PyGithub', 'openlabcmd']
```
Comment (wangxiyuan) [inline playbooks/ha_healthchecker.yaml]: `configparser` and `openlabcmd`  which have been installed in openlabcmd step can be removed as well.

Comment (bzhaoopenstack) [reply playbooks/ha_healthchecker.yaml]: Done

**Review by kiwik — APPROVED** [review]
LGTM, thank you.

**Comment by wangxiyuan** [comment]
Tested locally, works well. Thanks.

**Pull request merged**